### PR TITLE
New functions for Catalan and Euler-Mascheroni mathematical constants.

### PR DESCRIPTION
**Pull request by carandraug**
Adds two new functions for the symbolic package. I also have a python script to generate them automatically but in the end I thought that although quite simple, the script would be overkill.

### CODE REVIEW

File: inst/catalan.m [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+%% @seealso{eulergamma}
+%% @end deftypefn
+
+%% Author: Carnë Draug
+%% Keywords: symbolic, constants
+
+function g = catalan ()
+  g = python_cmd ('return sympy.S.Catalan,');
+end
+
```
Comment (cbm755) [inline inst/catalan.m]: trailing white space.

(Usually I want tests although I'm soon going to support doctests so maybe its overkill...)

File: inst/eulergamma.m [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @deftypefn {Function File} {} eulergamma ()
+%% Return Euler-Mascheroni constant.
+%%
+%% @example
+%% vpa (catalan ())
```
Comment (cbm755) [inline inst/eulergamma.m]: eulergamma

(and with every such error my belief in doctests grows stronger!)

File: inst/eulergamma.m [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+%% @seealso{catalan}
+%% @end deftypefn
+
+%% Author: Carnë Draug
+%% Keywords: symbolic, constants
+
+function g = eulergamma ()
+  g = python_cmd ('return sympy.S.EulerGamma,');
+end
+
```
Comment (cbm755) [inline inst/eulergamma.m]: trailing white space

**Comment by cbm755** [comment]
+1 after those changes.

**Comment by carandraug** [comment]
About the typo on documentation [here's the script](https://bpaste.net/show/16011b9109db) mentioned on the pull request (with this problem fixed) I guess it could be adjusted to fit with the other script you have on src/ but I didn't want to mess with your build system. Also, Matlab seems to only have this two functions, so unless you want to have the other constants in symbolic, there's even less reason for it.

I have also added two simple tests. I only started playing with the symbolic package this weekend (I was having fun implementing `psi` for Octave core) so I'm unsure what would be the best way.

I have also removed the last empty line as requested.

**Comment by cbm755** [comment]
Awesome thanks for this.  those tests should be fine.  This is in.

>  but I didn't want to mess with your build system.

That was very polite ;-)  But yes it is horrid.

I used to do code generation at install time, but now I've just checked the results in: slightly lowers barrier for new contributions (I hope).  

On the topic of build systems, is https://github.com/cbm755/octsympy/issues/196 an easy fix for you?

**Pull request merged**